src/main/java/frc/robot/subsystems/shooterPivot/ShooterPivot.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.subsystems.shooterPivot;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterPivot extends SubsystemBase {
+  private final ShooterPivotIO IO;
+
+  public ShooterPivot(ShooterPivotIO IO) {
+    this.IO = IO;
+  }
```

Comment:
This subsystem needs a period() method (see [this subsystem from last year](https://github.com/FRC2713/Robot2023/blob/9c3afcb173141d6d85c83b49ddad71a4b11eded1/src/main/java/frc/robot/subsystems/fourBarIO/FourBar.java#L125) as an example)